### PR TITLE
Fixup flipped axes in clustering confmat plots

### PR DESCRIPTION
**Pull request by plstcharles**
Silly mistake, does not modify code that anyone but @gmarceaucaron & I know about...

### CODE REVIEW

**Comment by plstcharles** [comment]
I can't merge this myself without 3rdparty approval, can someone click the lovely magic button?

**Review by obilaniu — APPROVED** [review]
LGTM

**Review by satyaog — APPROVED** [review]

**Comment by satyaog** [comment]
merging

**Pull request merged**
